
diff --git a/meson.build b/meson.build
index 515cc8501..5dd5764e8 100644
--- a/meson.build
+++ b/meson.build
@@ -63,7 +63,7 @@ dep_canberra = dependency('libcanberra')
 dep_canberra_gtk3 = dependency('libcanberra-gtk3')
 
 # Needed for window tracking
-dep_xfce4windowing = dependency('libxfce4windowing-0', version: '>= 4.20.4')
+dep_xfce4windowing = dependency('libxfce4windowing', version: '>= 4.20.4')
 
 # Needed for upower interaction
 dep_upower = dependency('upower-glib', version: '>= 0.99.8')
